src/sentry/static/sentry/app/components/actions/ignore.tsx

```diff
@@ -33,282 +27,257 @@ const IGNORE_WINDOWS: [number, string][] = [
-                          onAction={() => this.onIgnore({ignoreCount: count})}
+                          onAction={() =>
+                            onIgnore({
+                              ignoreCount: count,
+                              ignoreWindow: hours,
+                            })
+                          }
                         >
-                          {t('from now')}
+                          {label}
                         </ActionLink>
                       </MenuItem>
-                      {IGNORE_WINDOWS.map(([hours, label]) => (
-                        <MenuItem noAnchor key={hours}>
-                          <ActionLink
-                            {...actionLinkProps}
-                            onAction={() =>
-                              this.onIgnore({
-                                ignoreCount: count,
-                                ignoreWindow: hours,
-                              })
-                            }
-                          >
-                            {label}
-                          </ActionLink>
-                        </MenuItem>
-                      ))}
-                    </DropdownLink>
-                  </li>
-                ))}
-                <MenuItem divider />
-                <MenuItem noAnchor>
-                  <a onClick={() => this.setState({modal: ModalStates.COUNT})}>
-                    {t('Custom')}
-                  </a>
-                </MenuItem>
-              </DropdownLink>
-            </li>
-            <li className={submenuClassName}>
-              <DropdownLink
-                title={t('Until this affects an additional\u2026')}
-                caret={false}
-                isNestedDropdown
-                alwaysRenderMenu
-              >
-                {IGNORE_COUNTS.map(count => (
-                  <li className={submenuClassName} key={count}>
-                    <DropdownLink
-                      title={tn('one user\u2026', '%s users\u2026', count)}
-                      caret={false}
-                      isNestedDropdown
-                      alwaysRenderMenu
-                    >
-                      <MenuItem noAnchor>
+                    ))}
+                  </DropdownLink>
+                </li>
+              ))}
+              <MenuItem divider />
+              <MenuItem noAnchor>
+                <a onClick={openCustomIngoreCount}>{t('Custom')}</a>
+              </MenuItem>
+            </DropdownLink>
+          </li>
+          <li className={submenuClassName}>
+            <DropdownLink
+              title={t('Until this affects an additional\u2026')}
+              caret={false}
+              isNestedDropdown
+              alwaysRenderMenu
+            >
+              {IGNORE_COUNTS.map(count => (
+                <li className={submenuClassName} key={count}>
+                  <DropdownLink
+                    title={tn('one user\u2026', '%s users\u2026', count)}
```

Comment:
I wonder why this is ok, but the code on line 187-188 is different?

Reply:
😬